FEM/fem_ele_vec.cpp

```diff
@@ -2435,7 +2453,134 @@ void CFiniteElementVec::GlobalAssembly_RHS()
 				De->multi(strain_ne, dstress);
 				for (i = 0; i < ns; i++) // JT: This was commented. It shouldn't be.
 					dstrain[i] += strain_ne[i];
+		}
+			if (smat->Creep_mode == 1001) //BURGERS.
+			{
+			//get total strains at time t and current iteration i
+			double *strain_t, *strain_curr;
+			strain_t = new double[6]; strain_curr = new double[6];
+
+			//get stresses as well as internal variables at time t and current iteration i (equal before local Newton)
+			double *stress_curr, *eps_K_curr, *eps_M_curr;
+			stress_curr = new double[6]; eps_K_curr = new double[6]; eps_M_curr = new double[6];
+			for (int compnt (0); compnt<ns; compnt++){
+				strain_t[compnt] = (*eleV_DM->Strain_t_ip)(compnt,gp);
+				strain_curr[compnt] = strain_t[compnt] + dstrain[compnt];
+				stress_curr[compnt] = (*eleV_DM->Stress)(compnt, gp);
+				eps_K_curr[compnt] = (*eleV_DM->Strain_Kel)(compnt,gp);
+				eps_M_curr[compnt] = (*eleV_DM->Strain_Max)(compnt,gp);
+			}
+			//Fill remaining 3D components with 0 if only 2D
+			if (ns == 4)
+				for (int compnt (4); compnt<6; compnt++){
+					strain_t[compnt] = 0.0;
+					strain_curr[compnt] = 0.0;
+					stress_curr[compnt] = 0.0;
+					eps_K_curr[compnt] = 0.0;
+					eps_M_curr[compnt] = 0.0;
+				}
+
+			//6x6 tangent
+			Matrix* ConsD;
```

Comment:
no need to write `new`, since  you delete this pointer later. simply write `Matrix ConsD(6,6);`